proposals/low-level-struct-improvements.md

```diff
@@ -45,235 +42,125 @@ readonly ref struct Span<T>
 }
 ```
 
-The first challenge is the span safety rules make a [hard assumption](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.2/span-safety.md#span-constructor) that `Span<T>` has no such constructor. By extension this also introduced a hard assumption that `ref` fields do not exist. This resulted in a significant simplification of the rules but it allows for a number of patterns that make introducing `ref` fields significantly more difficult. 
-
-Consider the following method signature:
+The challenge in this proposal is the compatibility implications this design has to our existing [span safety rules](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/span-safety.md). While those rules fully support the concept of a `ref struct` having having `ref` fields they do not allow for APIs, other than `stackalloc`, to capture `ref` state that refers to the stack. The span safety rules have a [hard assumption](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.2/span-safety.md#span-constructor) that a constructor of the form `Span(ref T value)` does not exist. That means the safety rules do not account for a `ref` parameter being able to escape as a `ref` field hence it allows for code like the following.
```

Comment:
>having having [](http://example.com/codeflow?start=266&length=13)

Typo

Reply:
clearly i'm having having some issues here

Reply:
"having having" is still there. how come this is marked as resolved?

Reply:
It was removed in this commit which was pushed yesterday

https://github.com/dotnet/csharplang/pull/5959/commits/584743fd17aec94c2d46084bdf64e61df473008e

Are you not seeing that in your view?

Reply:
> It was removed in this commit which was pushed yesterday
> 
> [584743f](https://github.com/dotnet/csharplang/commit/584743fd17aec94c2d46084bdf64e61df473008e)
> 
> Are you not seeing that in your view?

I can see it in 584743f, and I thought it would show "here".  It turns out I was wrong. Sorry! my bad.